File: custom_components/meraki_ha/www/src/types/websocket.ts
export enum WsCommand {
  GET_CONFIG = 'meraki_ha/get_config',
  SUBSCRIBE_MERAKI_DATA = 'meraki_ha/subscribe_meraki_data',
  GET_CAMERA_STREAM_URL = 'meraki_ha/get_camera_stream_url',
  GET_CAMERA_SNAPSHOT = 'meraki_ha/get_camera_snapshot',
  GET_VERSION = 'meraki_ha/get_version',
  GET_NETWORK_EVENTS = 'meraki_ha/get_network_events',
  UPDATE_ENABLED_NETWORKS = 'meraki_ha/update_enabled_networks',
  CREATE_GUEST_KEY = 'meraki_ha/ipsk/create',
  GET_GUEST_KEYS = 'meraki_ha/ipsk/get',
  REVOKE_GUEST_KEY = 'meraki_ha/ipsk/revoke',
  TIMED_ACCESS_GET_POLICIES = 'meraki_ha/timed_access/get_policies',
}

export interface WsMessagePayload {
  type: string;
  [key: string]: any;
}

File: custom_components/meraki_ha/const/websocket.py
"""WebSocket command constants."""

from __future__ import annotations

from typing import Final

WS_CMD_GET_CONFIG: Final = "meraki_ha/get_config"
WS_CMD_SUBSCRIBE_MERAKI_DATA: Final = "meraki_ha/subscribe_meraki_data"
WS_CMD_GET_CAMERA_STREAM_URL: Final = "meraki_ha/get_camera_stream_url"
WS_CMD_GET_CAMERA_SNAPSHOT: Final = "meraki_ha/get_camera_snapshot"
WS_CMD_GET_VERSION: Final = "meraki_ha/get_version"
WS_CMD_GET_NETWORK_EVENTS: Final = "meraki_ha/get_network_events"
WS_CMD_UPDATE_OPTIONS: Final = "meraki_ha/update_options"
WS_CMD_UPDATE_ENABLED_NETWORKS: Final = "meraki_ha/update_enabled_networks"
WS_CMD_GET_GUEST_KEYS: Final = "meraki_ha/ipsk/get"
WS_CMD_CREATE_GUEST_KEY: Final = "meraki_ha/ipsk/create"
WS_CMD_REVOKE_GUEST_KEY: Final = "meraki_ha/ipsk/revoke"
WS_CMD_TIMED_ACCESS_GET_POLICIES: Final = "meraki_ha/timed_access/get_policies"
